components/pages/toast.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="author" content="Sharath Edulakanti">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Toast | Atom</title>
+    <link rel="stylesheet" href="../../css/main.css">
+    <script src="https://kit.fontawesome.com/0b01ac136a.js" crossorigin="anonymous">
+    </script>
+</head>
+
+<body>
+    <!-- nav bar -->
+    <nav class="navigation container">
+        <div class="nav-brand">
+            <a href="../../index.html" class="icon"><i class="fas fa-atom"></i></a>
+            Atom UI
+        </div>
+
+        <ul class="list-non-bullet nav-list">
+            <li class="list-item-inline">
+                <a class="link" href="../../index.html">Home</a>
+            </li>
+            <li class="list-item-inline">
+                <a class="link link-active" href="installation.html">Documentation</a>
+            </li>
+            <li class="list-item-inline">
+                <a class="link" href="https://github.com/sharath-io/Atom-UI" target="_blank">Github</a>
+            </li>
+        </ul>
+    </nav>
+
+    <!-- sidebar -->
+    <aside class="sidebar">
+        <ul>
+            <li><a href="installation.html">Installation</a></li>
+            <li><a href="avatar.html">Avatar</a></li>
+            <li><a href="badge.html">Badge</a></li>
+            <li><a href="alert.html">Alert</a></li>
+            <li><a href="button.html">Button</a></li>
+            <li><a href="card.html">Card</a></li>
+            <li><a href="image.html">Image</a></li>
+            <li><a href="input.html">Input</a></li>
+            <li><a href="navigation.html">Navigation</a></li>
+            <li><a href="rating.html">Rating</a></li>
+            <li><a href="slider.html">Slider</a></li>
+            <li><a href="toast.html">Toast</a></li>
+        </ul>
+    </aside>
+
+    <div class="body-text">
+        <h1>Toast</h1>
+        <p class="description">Toasts are lightweight notifications designed to mimic the push notifications that
+            have been popularized by mobile and desktop operating systems.Generally they are used as an interface
+            feedback pattern for informing the user about the results of an action.
+        </p>
+        <h2>Can check live demo of Toasts below</h2>
+
+        <button class="bt btn-primary" id="top-toast-id">Top Toast</button>
+        <button class="bt btn-primary" id="bottom-toast-id">Bottom Toast</button>
+
+        <div class="toast top-toast" id="top-content">
+            <p class="description toast-element"> This is top toast description
+                <span id="top-toast-close"><i class="fa fa-times"></i></span>
```

Comment:
use `buttton` element for close instead of `span`

Reply:
added button to close toast.